src/js/frontend/README.md

```diff
@@ -15,7 +15,7 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.\
 You will also see any lint errors in the console.
 
-### `yarn test`
+### `yarn integrationTest` or `yarn unitTest`
```

Comment:
I notice we seem to have a pattern of using `:` to create "subtypes" of a command (see `prettier` and `prettier:fix`; `lint` and `lint:fix`).

Any reason to not stick with that convention and do something like `test:integration` and `test:unit`?